charts/leonardo/values.yaml

```diff
@@ -115,3 +115,23 @@ zombieMonitorCron:
 vault:
   # vault.pathPrefix -- (string) Vault path prefix for secrets. Required if vault.enabled.
   pathPrefix:
+  # Migration credentials only referenced if migration.enabled == true
+  migration:
+    # vault.migration.path -- Vault path to secret containing DB credentials
+    path: null
+    # vault.migration.dbUsernameKey -- Key in Vault secret to DB username
+    dbUsernameKey: "db_user"
+    # vault.migration.DbPasswordKey -- Key in Vault secret to DB password
+    dbPasswordKey: "db_password"
+
+migration:
+  # migration.enabled -- (bool) Whether to run a Liquibase migration during sync
+  enabled: false
+  # migration.liquibaseCommand -- (string) Liquibase CLI command, can be "updateSQL" for a no-op dry-run
+  liquibaseCommand: "updateSQL"
```

Comment:
can you add a link to the doc what possible values are for this?

Reply:
Done, good idea. For normal usage this field might not be even exposed by default in the application values file--for non-testing purposes, the library chart's default of just always providing `update` (a normal, standard migration) would be sufficient, and as you mention above the `enabled` flag controls the behavior that most folks will want to access: whether liquibase runs or not.